### PR TITLE
bring examples in shape for examples.nutils.org

### PR DESCRIPTION
**Pull request by gertjanvanzwieten**
Includes backport of #696

### CODE REVIEW

File: examples/drivencavity.py [hunk 1]
```diff
@@ -1,104 +1,195 @@
...
+# tangential components are either strongly or weakly constrained depending on
+# the `strongbc` parameter. Since the tangential components at the top are
+# incompatible with the normal components at the left and right boundary, the
+# constraints are constructed in two steps to avoid Gibbs type oscillations,
+# and to make sure that the non-penetrating condition takes precedence.
+#
+# Depending on the `compatible` parameter, the script uses either a Taylor-Hood
+# (False) or a Raviart-Thomas (True) discretization. In case of TH, the system
+# is consistently modified by adding `∫_Ω .5 u_i v_i ∇_j(u_j)` to yield the
+# skew- symmetric advective term `∫_Ω .5 (v_i ∇_j(u_i) - u_i ∇_j(v_i)) u_j`. In
```
Comment (joostvanzwieten) [inline examples/drivencavity.py:54]: `skew- symmetric` -> `skew-symmetric`

File: examples/laplace.py [hunk 2]
```diff
@@ -1,24 +1,16 @@
-#! /usr/bin/env python3
+# Laplace problem on a unit square
 #
-# In this script we solve the Laplace equation :math:`u_{,kk} = 0` on a unit
-# square domain :math:`Ω` with boundary :math:`Γ`, subject to boundary
-# conditions:
+# In this script we solve the Laplace equation `Δu = 0` on a unit square domain
```
Comment (joostvanzwieten) [inline examples/laplace.py:3]: Laplace's equation (like Burgers')?

**Review by joostvanzwieten — CHANGES_REQUESTED** [review]

File: examples/cahnhilliard.py [hunk 3]
```diff
@@ -1,76 +1,85 @@
-#! /usr/bin/env python3
+# Unmixing of immiscible fluids
 #
 # In this script we solve the Cahn-Hilliard equation, which models the unmixing
-# of two phases (φ=+1 and φ=-1) under the influence of surface tension. It is a
-# mixed equation of two scalar equations for phase φ and chemical potential η::
+# of two phases (`φ=+1` and `φ=-1`) under the influence of surface tension. It
+# is a mixed equation of two scalar equations for phase φ and chemical
```
[the remark this answers is not in the document]
Comment (gertjanvanzwieten) [reply examples/cahnhilliard.py:5]: fixed

File: examples/laplace.py (hunk 2 above)
Comment (gertjanvanzwieten) [reply examples/laplace.py:3]: ok

**Review by joostvanzwieten — APPROVED** [review]

**Pull request merged**